src/main/java/bt/BT.java

```diff
@@ -179,15 +170,7 @@ public static void forgeBlock() {
      * Just for testing purposes.
      */
     public static void forgeBlock(String pass, int millis) {
-        Single<SubmitNonceResponse> submit = bns.submitNonce(pass, "0", null);
```

Comment:
I see you deleted the "sleep" part. That was for the testing functions, so that the local testnet can have time to actually process the block. Otherwise requests in sequence might not be processed yet and the tests fail. Is this now all handled by the submitNonce?

Reply:
Oops. Did not mean to remove sleep.